app/services/code_owners_service.rb

```diff
@@ -0,0 +1,34 @@
+class CodeOwnersService < BaseService
+  def call
+    Project.all.find_each do |project|
+      content_file = GithubReposApi.new(project.name).get_content_from_file('CODEOWNERS')
+      next if content_file.empty?
+
+      code_owners_names = build_code_owners_array(content_file)
+      remove_old_code_owners(code_owners_names, project)
+      add_new_code_owners(code_owners_names, project)
```

Comment:
Maybe we can benefit from using instance variables instead of passing around parameters, WDYT?

Reply:
could be a good idea, but that would mean having a new class. It wouldn't be a good idea to be setting instance variables inside a loop

Reply:
you're absolutely right! man I got so confused.